src/airgap/harness-airgap-images.sh

```diff
@@ -9,32 +9,56 @@ fail_count=0
 declare -a failed_images
 declare -a missing_images
 declare -a verified_images
+debug=false
+cleanup=false
+error_occurred=false
 
 # Function to display help message
 function show_help {
   echo "Usage: $0 -r <registry> [-f <tgz_file> | -d <tgz_directory>]"
   echo "  -r <registry>       The Artifactory registry name (e.g., artifactory.harness.internal/platform-staging)."
   echo "  -f <tgz_file>       The name of the .tgz file to process (optional if -d is provided)."
   echo "  -d <tgz_directory>  The directory containing .tgz files to process (optional if -f is provided)."
+  echo "  -D                  Enable debug mode for detailed logging."
+  echo "  -c                  Enable cleanup after script execution."
   exit 1
 }
 
 check_image_in_registry() {
   local image=$1
   if docker manifest inspect "$image" > /dev/null 2>&1; then
+    debug_log "Image $image is already present in the registry."
     return 0  # Image exists
   else
+    debug_log "Image $image is not present in the registry."
     return 1  # Image does not exist
   fi
 }
 
+debug_log() {
+  if [ "$debug" = true ]; then
+    echo "[DEBUG] $1"
+  fi
+}
+
+cleanup() {
+  echo "Cleaning up temporary files..."
+  if [ -n "$tgz_directory" ]; then
+    rm -rf "$tgz_directory"/*.tgz || debug_log "Failed to clean up .tgz files in directory."
```

Comment:
We shouldn't delete these files, cleanup should do docker cleanup, we can show the command we will run for cleanup and ask users for permissions. Deleting the downloaded files may cause issues where user may end up deleting the downloaded files and can't reuse them. Let's have these files cleanup manual

Reply:
This is optional, If user enabled flag with -c option then only it deletes it. I have seen CBP doing this due to space limitation.

Reply:
Will replace this to do docker cleanup